### ISSUE TITLE
broken link to normalization values

### ISSUE THREAD

**Issue by willtryagain** [pytorch/pytorch.github.io#1159]
## 📚 Documentation

There is a broken [link](https://pytorch.org/docs/master/torchvision/models.html) in the Load Data section of [this](https://pytorch.org/tutorials/beginner/finetuning_torchvision_models_tutorial.html) page.
